example/src/Examples/API/Images.tsx

```diff
@@ -26,10 +26,22 @@ const rects = [
 ];
 
 export const Images = () => {
+  // Verifies that the error handler for images are working correctly.
+  useImage(new Uint8Array([0, 0, 0, 255]), (err) =>
+    console.log(
+      err,
+      "UseImage failed when trying to read image data from a UInt8Array"
+    )
+  );
+  useImage("https://reactjs.org/invalid.jpg", (err) =>
+    console.log(
+      err,
+      "UseImage failed when trying to read image data from an uri"
+    )
```

Comment:
in the log message, shouldn't we mention that this is an expected error?

Reply:
I changed this one fail if the error message was wrong.